source/vkopt.js

```diff
@@ -2996,6 +2996,9 @@ vkopt['photoview'] =  {
             <a target="_blank" class="pv_more_act_item fl_l vk_ph_copy_search" href="https://www.google.com/searchbyimage?image_url={vals.src}">Google</a>
             <a target="_blank" class="pv_more_act_item fl_l vk_ph_copy_search" href="http://www.tineye.com/search?url={vals.src}">TinEye</a>
             <a target="_blank" class="pv_more_act_item fl_l vk_ph_copy_search" href="http://yandex.ru/images/search?img_url={vals.src}&rpt=imageview">Yandex</a>
+            <a target="_blank" class="pv_more_act_item fl_l vk_ph_copy_search" href="https://www.bing.com/images/searchbyimage?cbir=sbi&imgurl={vals.src}">Bing</a>
+            <a target="_blank" class="pv_more_act_item fl_l vk_ph_copy_search" href="http://karmadecay.com/{vals.src}">KarmaDecay</a>
+            <a target="_blank" class="pv_more_act_item fl_l vk_ph_copy_search" href="http://imgops.com/{vals.src}">ImgOps</a>

```

Comment:
```suggestion
            <a target="_blank" class="pv_more_act_item fl_l vk_ph_copy_search" href="https://imgops.com/{vals.src}">ImgOps</a>
```
ImgOps has valid TLS cert :cat2: